vms/platformvm/state/metadata_delegator.go

```diff
@@ -9,17 +9,35 @@ import (
 )
 
 type delegatorMetadata struct {
-	PotentialReward uint64
+	PotentialReward uint64 `v0:"true"`
+	StakerStartTime int64  `          v1:"true"`
```

Comment:
why is this int64?

Reply:
it's the type returned by time.Unix() method

Reply:
I don't think it is necessary. I can't recall why `LastUpdated` was typed as `uint64` in the first place. Currently we cast it to and from `int64` in order to convert it to a `time.Time`. I think we can avoid the type cast for `StakerStartTime`